content/applications/websites/livechat.rst

```diff
@@ -150,59 +172,68 @@ Fill out the fields on the :guilabel:`Open: Rules` pop-up as instructed below, t
+      :guilabel:`URL Regex` field, input the relative URL of the page where the chat button should
+      appear.
 
-   .. tab:: Open automatically timer
+      For example, to apply the rule to the URL, `https://mydatabse.odoo.com/shop`, enter `/shop`
+      to the :guilabel:`URL Regex` field.
 
-      This field designates the amount of time (in seconds) a page will be open before the chat
-      window will open. If the :guilabel:`Livechat Button` for this rule is not set to
-      :guilabel:`Open automatically`, this field will be ignored.
+      To apply the rule to *all* pages on the database, enter `/` in the :guilabel:`URL Regex`
+      field.
 
+   .. tab:: Open automatically timer
+
+      This field designates the amount of time (in seconds) a page should be open before the chat
+      window opens. This field **only** appears if the :guilabel:`Live Chat Button` for this rule is
+      set to :guilabel:`Open automatically`.
    .. tab:: Country
 
       If this channel should only be available to site visitors in specific countries, add them to
-      the :guilabel:`Country` field. If this field is left blank, the channel will be available to
-      all site visitors, regardless of location.
+      the :guilabel:`Country` field. If this field is left blank, the channel is available to all
+      site visitors, regardless of location.
 
 .. note::
-   In order to track the geographical location of visitors, :guilabel:`GeoIP` must be installed on
-   the database. While this feature is installed by default on *Odoo Online*, *On-Premise* databases
-   will require additional :doc:`setup steps </administration/on_premise/geo_ip>`.
+   To track the geographical location of visitors, *GeoIP* **must** be installed on the database.
+   While this feature is installed by default on *Odoo Online* databases, *On-Premise* databases
+   require additional :doc:`setup steps </administration/on_premise/geo_ip>`.
+
+.. _livechat/widget-tab:
 
 Widget
 ------
 
-The :guilabel:`Widget` tab on the live chat channel details form provides the shortcode for an
-embeddable website widget. This code can be added to a website to provide access to a live chat
-window.
+The :guilabel:`Widget` tab on the live chat channel detail form provides the code for a website
+widget. This code can be added to a website to provide access to a live chat window.
 
-The live chat widget can be added to websites created through Odoo by navigating to the
-:menuselection:`Website --> Configuration --> Settings`. Then scroll to the :guilabel:`Livechat`
-section, and select the channel to add to the site. Click :guilabel:`Save` to apply.
+.. tip::
+   The live chat widget can be added to websites created through Odoo by navigating to
+   :menuselection:`Website app --> Configuration --> Settings`. Then, scroll to the
+   :menuselection:`Email & Marketing` section. In the :guilabel:`Channel` field, select the channel
+   to add to the site. Click :guilabel:`Save` to apply.
```

Comment:
This 'Email & Marketing' section, nor the 'Channel' field are present in the 16.0 runbot I'm using for the review. The closest field I found was a 'Channel' field, beneath the LiveChat setting, in the 'How to configure the Twitter API access'... But I don't believe this is the same thing. Please double-check the accuracy of this statement during your revisions.

Reply:
It is the same section, the changed the heading (I believe in 16.2 or 16.3). I have  made the update to reflect the UI